cypress/e2e/navbar&menu_burger/navbar_icon.cy.js

```diff
@@ -6,7 +6,7 @@ describe("Navbar icons", function () {
     });
     beforeEach(function () {
         cy.visit(this.testData.url_laser);
-        cy.wait(10000);
+        cy.wait(20000);
```

Comment:
Can'T we just wait for a element to be visible with a timepout, this will speed up the tests if it will show faster